bigtable/google/cloud/bigtable/cluster.py

```diff
@@ -42,17 +46,35 @@ class Cluster(object):
     :type instance: :class:`~google.cloud.bigtable.instance.Instance`
     :param instance: The instance where the cluster resides.
 
+    :type location_id: str
+    :param location_id: ID of the location in which the cluster will be
+                        created. For e.g "us-central1-b"
+                        List of locations id is :
+                        https://cloud.google.com/bigtable/docs/locations
+                        For best performance, clients should be located
+                        as close as possible to this cluster.
+
     :type serve_nodes: int
     :param serve_nodes: (Optional) The number of nodes in the cluster.
                         Defaults to :data:`DEFAULT_SERVE_NODES`.
+
+    :type default_storage_type: int
+    :param default_storage_type: (Optional) The default values are
```

Comment:
This parameter is actually named `storage_type` below.

Reply:
Done.